### ISSUE TITLE
missing node_modules/caniuse-lite/data/features/css-unicode-bidi

### ISSUE THREAD

**Issue by Rongeegee** [vercel/next.js#40461]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

Operating System:
      Platform: linux
      Arch: x64
      Version: #141~18.04.1-Ubuntu SMP Thu Aug 11 20:15:56 UTC 2022
    Binaries:
      Node: 16.13.1
      npm: 8.19.1
      Yarn: 1.22.19
      pnpm: 7.11.0
    Relevant packages:
      next: 12.3.0
      eslint-config-next: 12.3.0
      react: 18.2.0
      react-dom: 18.2.0

warn  - Latest canary version not detected, detected: "12.3.0", newest: "12.3.1-canary.0".
        Please try the latest canary version (`npm install next@canary`) to confirm the issue still exists before creating a new issue.
        Read more - https://nextjs.org/docs/messages/opening-an-issue

### What browser are you using? (if relevant)

Chrome

### Describe the Bug

i created a Next Project this command

`npx create-next-app@lates`

When I try to run the project with

`npm run dev'

I got this error:
```
error - ./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[2].oneOf[8].use[1]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[2].oneOf[8].use[2]!./styles/globals.css
Error: Cannot find module 'caniuse-lite/data/features/css-unicode-bidi'
```

I fixed this issue by manually moving css-unicode-bidi.js from project of older version to node_modules/caniuse-lite/data/features

### Expected Behavior

css-unicode-bidi.js should be created in  node_modules/caniuse-lite/data/features

### Link to reproduction

n/a

### To Reproduce

just create a next app with npx create-next-app@latest, you will see

**Comment by marsidev** [comment]
related https://github.com/browserslist/caniuse-lite/issues/102

**Comment by priyansh32** [comment]
It has been fixed in caniuse-lite [#Update caniuse-db 1.0.30001397](https://github.com/browserslist/caniuse-lite/commit/62622e00b26f96992dc0de08c4e4bb3a636ae608) It should work now.

**Comment by balazsorban44** [comment]
As pointed out above, it was an upstream issue, so I'm closing. Thanks for reporting anyway!
